### PR TITLE
fixed summary page

### PR DESCRIPTION
**Pull request by DecathectZero**
http://i.imgur.com/ZY8dKpA.gif - This is the principle I went with for redesigning the summary marks table.

made it so that when view old marks the total is shown in yellow as visual indicator. 
Removed extra_mark_percentages showing up twice.

Bonus marks do not show if there are none.

Differences:
screen shot 2017-05-10 at 10 40 09 am
screen shot 2017-05-10 at 10 39 59 am

### CODE REVIEW

File: app/views/results/student/_student_summary.html.erb [hunk 1]
```diff
@@ -135,24 +139,10 @@
       <strong><%= t('marker.marks.translated_to_points') %></strong>
       <span id='total_extra_percentage_as_points'><%= result.get_total_extra_percentage_as_points %></span><br>
     </div>
+    <% end %>
   <% end %>
 </div>
 
-<% unless @result.is_a_review? %>
```
Comment (DecathectZero) [inline app/views/results/student/_student_summary.html.erb]: This entire section is a repeat of lines 130 - 141

File: app/assets/stylesheets/common/_constants.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
            $fonts: 'Open Sans', 'Lucida Grande', Verdana, Arial, sans-serif;
       $mono-fonts: 'Consolas', 'Courier New', Courier, mono, serif;
-          $radius: 5px;
```
Comment (DecathectZero) [inline app/assets/stylesheets/common/_constants.scss]: Preference: Markus looks better when it is angular and has sharper, square edges.

Comment (david-yz-liu) [inline app/assets/stylesheets/common/_constants.scss]: Please revert this change.

File: app/assets/stylesheets/grader.scss [hunk 3]
```diff
@@ -882,27 +882,29 @@ ul.annotation_text_list {
   padding: 0;
   width: 100%;
 
+  thead{
```
Comment (david-yz-liu) [inline app/assets/stylesheets/grader.scss]: Put a space before the `{` (occurs below as well)

File: app/views/results/student/_student_summary.html.erb [hunk 4]
```diff
@@ -26,6 +26,7 @@
       </div>
     </div>
 
+    <% if (@old_result.get_positive_extra_points != 0 or @old_result.get_negative_extra_points != 0) %>
```
Comment (david-yz-liu) [inline app/views/results/student/_student_summary.html.erb]: Don't use parentheses around the condition. Also, use `||` instead of `or` (these are actually different).

File: app/views/results/student/_student_summary.html.erb (hunk 1 above)
Comment (david-yz-liu) [inline app/views/results/student/_student_summary.html.erb]: It's fine to remove the contents here, but then the condition `@result.is_a_review?` should be included above.

Comment (DecathectZero) [inline app/views/results/student/_student_summary.html.erb]: The entire block in the above section is already wrapped inside a @result.is_a_review? condition
